### PR TITLE
WIP: Automatically pull PO files from POEditor

### PR DESCRIPTION
**Pull request by kimadactyl**

### CODE REVIEW

File: locale/ar/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -105,27 +105,27 @@ msgid "Welcome to %(server)s, %(member name)s!\n"
 "INFO: Get this message again\n"
 "\n"
 "You can also message me on Telegram: %(telegram link)s"
-msgstr "مرحبا بك في member name)s! ,%(server)s)%\n"
+msgstr " مرحبا %(s(member name في, %(s(server\n"
```
Comment (WheresAlice) [inline locale/ar/LC_MESSAGES/django.po:108]: This looks wrong.

If you merge my PRs you'll get additional locale tests that catch broken tokens like this.

File: locale/ar/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -105,27 +105,27 @@ msgid "Welcome to %(server)s, %(member name)s!\n"
...
 "\n"
-"يمكنك أن ترسل(ي) لي طلبات التالية, او ارسل(ي) رسالة إلى :imok phone number)s)%\n"
+"يمكنك أن ترسل(ي) لي طلبات التالية, او ارسل(ي) رسالة إلى %(imok phone number)s:\n"
 "\n"
-":IN وقت الوصول إلى المركز signing center)s)%\n"
+" ارسل(ي)  IN  عند وصولك(ي) إلى مركز %(s(signing center\n"
```
Comment (WheresAlice) [inline locale/ar/LC_MESSAGES/django.po:112]: this looks broken

File: locale/ar/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -105,27 +105,27 @@ msgid "Welcome to %(server)s, %(member name)s!\n"
...
+"لتحديث اسمك(ي)  ارسل(ي)   NAME  مصحوب بالإسم\n"
 "\n"
-":SOS أرفع(ي) جرس الإنذار\n"
+"في حالة طوارء ارسل(ي)  SOS  لرفع جرس الإنذار\n"
 "\n"
-":INFO احصل(ي) على هذه الرسالة مرة أخرى\n"
+"لتتسلم هذه  الرسالة مجددا ارسل(ي)   INFO احصل(ي) \n"
 "\n"
-"يمكنك أيضًا مراسلتي على تلغرام :telegram link)s)%"
+"يمكنك أيضًا مراسلتي عبر تطبيق تلغرام : %(s(telegram link"
```
Comment (WheresAlice) [inline locale/ar/LC_MESSAGES/django.po:122]: this looks broken

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -1,166 +1,132 @@
...
-"Your check out time from %(center)s is %(time)s.\n"
+msgid "Your check out time from %(center)s is %(time)s.\n"
 "\n"
 "I hope you have a lovely day!"
-msgstr ""
-"Vous quittez le centre %(center)s à %(time)s.\n"
+msgstr "Vous avez quitté le centre %(center)s à %(time)s.\n"
 "\n"
-"Nous vous souhaitons une merveilleuse journée!"
+"Nous vous souhaitons une bonne journée! "
```
Comment (WheresAlice) [inline locale/fr_FR/LC_MESSAGES/django.po:63]: there's an extra space here

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request closed without merging**